dev-java/jakarta-activation-api/jakarta-activation-api-1.2.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom activation/pom.xml --download-uri https://github.com/eclipse-ee4j/jaf/archive/refs/tags/1.2.2.tar.gz --slot 1 --keywords "~amd64 ~arm ~arm64 ~ppc64 ~x86" --ebuild jakarta-activation-1.2.2.ebuild
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="com.sun.activation:jakarta.activation:1.2.2"
```

Comment:
i suppose this should be `jakarta-activation-api`.

Reply:
> i suppose this should be `jakarta-activation-api`.

Confusion derived from different `pom.xml` files. I regenerated the `ebuild` with `java-ebuilder` from [jaf/activationapi/pom.xml](https://github.com/eclipse-ee4j/jaf/tree/e199f61f414fa3455734a407a6e56469b3ad97b4/activationapi) and also adjusted `SRC_URI`.